src/Resources/Locales/de_DE.axaml

```diff
@@ -407,7 +408,7 @@
   <x:String x:Key="Text.Preference.GPG.UserKey.Placeholder" xml:space="preserve">GPG Benutzer Signierungsschlüssel</x:String>
   <x:String x:Key="Text.Preference.DiffMerge" xml:space="preserve">DIFF/MERGE TOOL</x:String>
   <x:String x:Key="Text.Preference.DiffMerge.Path" xml:space="preserve">Installationspfad</x:String>
-  <x:String x:Key="Text.Preference.DiffMerge.Path.Placeholder" xml:space="preserve">Gebe Installationspfad von Diff/Merge Tool an</x:String>
+  <x:String x:Key="Text.Preference.DiffMerge.Path.Placeholder" xml:space="preserve">Gebe Installationspfad zum Diff/Merge Tool an</x:String>
```

Comment:
Phew, that's tricky. I think both are valid. Just ensure consistency.

Reply:
Actually I changed my mind. I think just "Installationspfad zum Diff/Merge Tool" is better since the longer version doesn't fit in the space:
![grafik](https://github.com/user-attachments/assets/5ce0b6ba-cf78-4c60-aefe-9bbc9bcf62e3)

Yes this is different from the english translation but should at least fit. Will also change the other text.